scrapy/commands/check.py

```diff
@@ -9,7 +9,7 @@
 
 
 class TextTestResult(_TextTestResult):
-    def printSummary(self, start, stop):
+    def print_summary(self, start, stop):
```

Comment:
This follows the base class naming convention.

Reply:
So, the class naming convention supersedes the PEP8?

Reply:
I didn't mean that, just an observation.

Reply:
understood. thank you